packages/client-sdk-nodejs/test/unit/config/transport/transport-strategy.test.ts

```diff
@@ -39,9 +41,11 @@ describe('StaticGrpcConfiguration', () => {
 describe('StaticTransportStrategy', () => {
   const testDeadlineMillis = 90210;
   const testMaxSessionMemoryMb = 90211;
+  const testNumClients = 6;
```

Comment:
let's use a very unique value here so that we can be certain the test is not accidentally passing due to the test value matching some default value.